articles/virtual-desktop/deploy-azure-ad-joined-vm.md

```diff
@@ -43,10 +50,13 @@ You can deploy Azure AD-joined VMs directly from the Azure portal when [creating
 > - Host pools should only contain VMs of the same domain join type. For example, AD-joined VMs should only be with other AD VMs, and vice-versa.
 > - The host pool VMs must be Windows 10 single-session or multi-session, version 2004 or later.
 
-After you've created the host pool, you must assign user access. For Azure AD-joined VMs, you'll need to do two things:
+### Assign user access to host pools
+After you've created your host pool, you must assign users access to let them access to their resources. To grant access to resources, add each user to the app group. You'll need to do this process for both Azure Virtual Desktop and Azure Virtual Desktop (classic) deployments.
 
-- Add users to the App Group to give them access to the resources.
-- Grant users the Virtual Machine User Login role so they can sign in to the VMs.
+For Azure AD-joined VMs, you'll need to do three extra things on top of the requirements for Active Directory or Azure Active Directory Domain Services-based deployments:  
```

Comment:
```suggestion
For Azure AD-joined VMs, you'll need to do two extra things on top of the requirements for Active Directory or Azure Active Directory Domain Services-based deployments:  
```

Reply:
I'm thinking there might be some older data in what you are seeing here.   This is what I currently have which is 3 bullets.  I'll get the other updates noted and ensure I have pushed the changes up from my local machine.  

For Azure AD-joined VMs, you'll need to do three extra things on top of the requirements for Active Directory or Azure Active Directory Domain Services-based deployments:  
- Assign your users the **Virtual Machine User Login** role so they can sign in to the VMs.
- Assign administrators who need local administrative privileges the **Virtual Machine Administrator Login** role.
- To access resources from client machines or endpoint devices that aren't Azure AD-joined to your tenant or are outside your organization, you'll need to add a new property called *targetisaadjoined:i:1* to your host pool. To add this property, open the **Advanced** tab, then enter the property under **RDP Properties**.